### PR TITLE
Add unix shell scripts for (re)building databases

### PR DESCRIPTION
**Pull request by tnleeuw**
Add shell scripts for databases -- only for the databases for now as that's what I need most, we can add similar scripts for other images later I think.

### CODE REVIEW

**Comment by philipsens** [comment]
We could also ditch a lot of these scripts in favor of docker compose, like we did in IAF. (https://github.com/ibissource/iaf/tree/master/docker)

**Comment by tnleeuw** [comment]
> We could also ditch a lot of these scripts in favor of docker compose, like we did in IAF. (https://github.com/ibissource/iaf/tree/master/docker)

The scripts help building the images with use of docker-compose though, right?

In IAF we use Maven for that.

File: docker/dbms/MySQL/Dockerfile [hunk 1]
```diff
@@ -9,7 +9,4 @@ ENV TZ=Europe/Amsterdam \
 
 COPY create_user.sql /docker-entrypoint-initdb.d/
 
-# Change port so mysql can be run together with mariadb
-CMD ["mysqld", "--port", "3307"]
-
-EXPOSE 3307
```
[the remark this answers is not in the document]
Comment (tnleeuw) [reply docker/dbms/MySQL/Dockerfile]: Fixed!

**Comment by tnleeuw** [comment]
@nielsm5 @philipsens Zouden jullie dit PR alsjeblieft willen approven en mergen? Jacobjob gaat deze scripts binnenkort ook nodig hebben om images te kunnen bouwen voor het draaien van IAF-Test.

**Comment by philipsens** [comment]
> > We could also ditch a lot of these scripts in favor of docker compose, like we did in IAF. (https://github.com/ibissource/iaf/tree/master/docker)
> 
> The scripts help building the images with use of docker-compose though, right?
> 
> In IAF we use Maven for that.

In IAF we use Maven to get the dependencies and then run a Docker build commando e.g. `docker build . -t name` 
The compose files in IAF are used to run the images. 

This repo only has Dockerfiles and doesn't need to gather any dependencies. So only using docker compose would suffice. 
We also build the images with Jenkins and put them on our private Nexus. So if someone doesn't have special needs and just wants to run a database, they can run `docker compose up` and the image will be downloaded. 
In the case that somebody does want to change something, they can build the image using `docker compose build`.

I dislike the idea of using scripts for such a simple task. The commands aren't complicated, they are documented [here](https://github.com/ibissource/iaf-ci-images#usage) and [here](https://docs.docker.com/compose/reference/), and it relies on existing tooling that people already have without needing to maintain multiple scripts (not to mention that the bat files were removed from `dbms` for this reason recently). 

Are you maybe planning on using the scripts for something more advanced in the future?

**Comment by tnleeuw** [comment]
> > > We could also ditch a lot of these scripts in favor of docker compose, like we did in IAF. (https://github.com/ibissource/iaf/tree/master/docker)
> > 
> > 
> > The scripts help building the images with use of docker-compose though, right?
> > In IAF we use Maven for that.
> 
> In IAF we use Maven to get the dependencies and then run a Docker build commando e.g. `docker build . -t name` The compose files in IAF are used to run the images.
> 
> This repo only has Dockerfiles and doesn't need to gather any dependencies. So only using docker compose would suffice. We also build the images with Jenkins and put them on our private Nexus. So if someone doesn't have special needs and just wants to run a database, they can run `docker compose up` and the image will be downloaded. In the case that somebody does want to change something, they can build the image using `docker compose build`.
> 
> I dislike the idea of using scripts for such a simple task. The commands aren't complicated, they are documented [here](https://github.com/ibissource/iaf-ci-images#usage) and [here](https://docs.docker.com/compose/reference/), and it relies on existing tooling that people already have without needing to maintain multiple scripts (not to mention that the bat files were removed from `dbms` for this reason recently).
> 
> Are you maybe planning on using the scripts for something more advanced in the future?

When I added the unix shell scripts, the DOS batch files were still there and they were a convenient way to rebuild the images and start them, but I needed unix versions and not DOS versions.
At the time DB2 did not have a compose file yet either so the scripts made it easier.

Starting an image is just "docker-compose up -d" so that is of course easy, for rebuilding I liked the convenience of having the script bring the service down and removing all volumes etc for me.

But hey I can keep the shell scripts local to myself if you rather not merge these!
At the time I did them I did not know the plan was to delete the scripts altogether.

**Pull request closed without merging**